cppseminar/docker-compose.yml

```diff
@@ -103,6 +103,26 @@ services:
           test: ['CMD', 'curl', '-f', 'http://localhost/healthcheck']
           timeout: 10s
 
+    mqreadservice.local:
+        build:
+            context: ./mqreadservice
+            target: dev
+        networks:
+            - sharednet
+        volumes:
+            - ./mqreadservice/Program.cs:/src/Program.cs
+            - ./mqreadservice/Controllers:/src/Controllers
+            - ./mqreadservice/Models:/src/Models
+            - ./mqreadservice/Helpers:/src/Helpers
+        ports:
+            - 8085:80
+        environment:
+            LOG_PRETTY: 1
+            TEST_INPUT_QUEUE: test-input
+            MQ_URI: amqp://rabbitmq.local/
+            VM_TEST_RETURN_ADDR: ${VM_TEST_RETURN_ADDR}:8084
```

Comment:
Cannot you use the name of the service directly?

Reply:
Or maybe it is fine, but in that case, thera are some leftovers in `vmtestserver.local:`.

Reply:
For now I leaving it as it is.